paimon-web-ui-new/src/views/metadata/components/breadcrumb/index.tsx

```diff
@@ -0,0 +1,35 @@
+/* Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License. */
+
+import { Folder } from '@vicons/ionicons5'
+
+export default defineComponent({
+  name: 'MetaDataBreadcrumb',
+  setup() { },
+  render() {
+    return (
+      <n-breadcrumb>
+        <n-breadcrumb-item>
+          <n-icon component={Folder} /> 北京总行</n-breadcrumb-item>
```

Comment:
Why use these banks?

Reply:
It's just a placeholder, it will become the folder name after the api is integrated.